### PR TITLE
docs: add Aristeidis Bampakos to contributor list

### PR DESCRIPTION
**Pull request by bampakoa**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -901,5 +901,13 @@
     "bio": "Teri is a technical editor working on the Angular docs team.",
     "groups": ["Angular"],
     "lead": "aikidave"
+  },
+  "abampakos": {
```
Comment (petebacondarwin) [inline aio/content/marketing/contributors.json:905]: I believe that this is supposed to be your Github username. But from it looks like yours is actually https://github.com/bampakoa

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
LGTM once the Github username is fixed.

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -901,5 +901,13 @@
...
+    "name": "Aristeidis Bampakos",
+    "picture": "aristeidis.jpg",
+    "twitter": "abampakos",
+    "website": "https://medium.com/@abampakos",
+    "bio": "Aristeidis is a front end web developer at Plex-Earth working with Angular. He is a co-organizer of the Angular Athens meetup and the author of Learning Angular - 3rd Edition. He is also a senior Angular instructor at Code.Hub where he teaches the Angular framework to other developers",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json:910]:
```suggestion
    "bio": "Aristeidis is a front end web developer at Plex-Earth working with Angular. He is a co-organizer of the Angular Athens meetup and the author of Learning Angular - 3rd Edition. He is also a senior Angular instructor at Code.Hub where he teaches the Angular framework to other developers.",
```

😇

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Review by mgechev — APPROVED** [review]

**Review by Splaktar — APPROVED** [review]
LGTM

**Pull request closed without merging**